raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Liquid waste disposal 
+(126) The local government authorities shall designate and ensure compliance with designated disposal ponds, sewage treatment facilities and sewer points.
+
+Control and monitoring of sewerage system 
+(127) Upon completion of construction of a sewerage system, local government authorities shall facilitate the carrying out of initial and subsequent periodic tests to ascertain that the effluent meant for final disposal meets the standards as may be prescribed by the Minister.
+
+Industrial liquid waste 
+(128) The Minister shall make regulations prescribing the best method for the treatment of hazardous and non-hazardous liquid wastes from industries. 
+
+Storm water management 
+(129) [1] Every local government authority shall, within its area of jurisdiction, build or prepare storm water drains. [2] The storm water drains shall remain clean for the purposes for which they have been made. [3] The Minister may make rules providing for: [a] standard gradient for storm water drains to prevent water stagnation; [b] the periodic cleaning of storm water drains to remove deposits; and [c] covered storm water drains, inspection covers and appropriate trap chambers for inspection and removal of deposits. 
+
+[d] Management of Gaseous Waste
+
+Gaseous wastes from dwelling houses 
+(130) The local government authorities shall ensure that all habitable buildings: [a] have adequate openings to provide through or cross ventilation; [b] provide adequate means of smoke escape in kitchen and other indoor places as may be appropriate; and [c] do not provide for cooking in dwelling houses or use of animal dung as fuel in confined unventilated dwellings. 
+
+Control of industrial gaseous waste 
+(131) [1] The local government authorities shall, taking into consideration of the wind direction and environmental impact assessment recommendations, ensure that industries producing gas emissions are located far away from residential areas. [2] Each local government authority shall prescribe rules, procedures and penalties governing: [a] regularity of checks of the machinery to ensure that they function efficiently and do not produce gas emissions; and [b] specifications of chimneys and gas controlling devices to be fitted to factories and industries. 
+
+Gaseous and particulate wastes from vehicles 
+(132) [1] The local government authorities shall prescribe rules specifying emission of carbon monoxide, hydrocarbons, other noxious emissions and standards for exhaust emissions applicable in areas of jurisdiction of a local government authority concerned. [2] Where national standards have been prescribed, the local government authorities shall adopt the national exhaust emissions standards. [e] Management of Hazardous Waste 
```

Comment:
Please replace `[e]` with `(e)` and add two newline characters before the header.